.gitignore

```diff
@@ -0,0 +1,17 @@
+# OSX leaves these everywhere on SMB shares
```

Comment:
We should have a .gitignore only at the root and not in each subdir, please

Reply:
Isn't that at the root? There is no .gitignore at all at the moment. Maybe I am just not getting your comment? :-)